unity/CAVE/Assets/Resources/CLI.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.IO;
+using System.Xml;
+using System.Text.RegularExpressions;
+using UnityEngine;
+using UnityEditor.SceneManagement;
+using UnityEditor.SceneTemplate;
+
+public class CLI : MonoBehaviour
+{
+    static void Main()
+    {
+        string xmlPath = null;
+
+        // Get command line arguments from Python
+        try {
+            string[] args = System.Environment.GetCommandLineArgs();
```

Comment:
You can just use `var` instead of specifying types, just like you'd use `auto` in C++